### DISCUSSION TITLE
Theme Previewer modifying the actual theme in the config

### DISCUSSION THREAD

**Discussion by chaezwav** [ghostty-org/ghostty#2802] [Feature Requests, Ideas] [answered]
If I ever run `ghostty +list-themes`, I get a nifty little preview pane for all the themes that I can choose from and write into my config file. I think it'd be a nice qol feature if we could just hit enter on the theme we'd like and it'd update.

![image](https://github.com/user-attachments/assets/c2e259a3-dbfe-45df-8c0a-94e59a9168f0)

Currently if I hit enter on anything, it doesn't do anything for me.

**Comment by 00-kat** [comment]
#2353 is probably a prerequisite for this, since there's currently no plumbing to control Ghostty externally.

**Comment by rhodes-b** [comment, marked as answer]
This is a duplicate and tip now displays a pop saying you need to add it to your config manually but like kat said above this would require #2353
